src/frontend/src/common/components/library/data_subview/components/DownloadModal/style.ts

```diff
@@ -38,43 +39,52 @@ export const Title = styled.span`
   }}
 `;
 
+const checkBoxInfoSpacing = (props: CommonThemeProps) => {
+  const spaces = getSpaces(props);
+  return `
+    position: inline-block;
+    float: left;
```

Comment:
you can feel free to change `float`s to `flex`s if you feel like it! (but no pressure 💪)